src/pages/admin/[cid]/index.tsx

```diff
@@ -0,0 +1,135 @@
+import { Label } from '@components/basic/Label'
+import { RootState } from '@redux/store'
+import { useRouter } from 'next/router'
+import { useCallback, useEffect, useState } from 'react'
+import { useSelector } from 'react-redux'
+import { request } from '@utils/api'
+import { LoadClassInfoParams, LoadClassInfoResults } from '@api/admin/loadClassInfo'
+import styled from '@emotion/styled'
+import { palette, typography } from '@styles/theme'
+import { TABLET_WIDTH_THRESHOLD } from 'src/constants/ui'
+import { FillBtn } from '@components/basic/button/Button'
+
+export default function Page() {
+  const isAdmin = useSelector((state: RootState) => state.userInfo.isAdmin)
+  const { push, query } = useRouter()
+  const cid = query.cid as string
+  const [classInfo, setClassInfo] = useState<LoadClassInfoResults>()
+
+  useEffect(() => {
+    !isAdmin
```

Comment:
I suggest to use `if` and `else` for better readability. Ternary operators are more appropriate when you need the return value.

Reply:
> I suggest to use `if` and `else` for better readability. Ternary operators are more appropriate when you need the return value.

Oh ok, I have updated it that way